hack/test/e2e-qemu.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
we probably want to update this with the latest fixes we have in `e2e-firecracker.sh` (for the registry at least)...